packages/devextreme-angular/README.md

```diff
@@ -1,26 +1,27 @@
-[![CircleCI](https://img.shields.io/circleci/project/github/DevExpress/devextreme-angular/master.svg)](https://circleci.com/gh/DevExpress/devextreme-angular) [![npm version](https://badge.fury.io/js/devextreme-angular.svg)](https://badge.fury.io/js/devextreme-angular)
+# DevExtreme for Angluar
 
+[![NPM version](https://img.shields.io/npm/v/devextreme-angular.svg?maxAge=43200)](https://www.npmjs.com/package/devextreme-angular)
 
-# DevExtreme Angular UI Components #
+The `devextreme-angular` package contains the [DevExtreme component suite](http://js.devexpress.com/Demos/WidgetsGallery/) for the [Angular](https://angular.io/) framework.
 
-This project allows you to use [DevExtreme widgets](http://js.devexpress.com/Demos/WidgetsGallery/) in [Angular](https://angular.io/) applications.
+* [Documentation](https://js.devexpress.com/Angular/Documentation/Guide/Angular_Components/DevExtreme_Angular_Components/)
+* [Component Demos](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/Overview/Angular/Light/)
+* **Template Gallery**
+    * [Fluent theme](https://devexpress.github.io/devextreme-ui-template-gallery/fluent.blue.light/angular/default/#/crm-contact-list)
```

Comment:
There is no a link for Material
But I think it's better to give a link to the Angular UITG Landing https://js.devexpress.com/Angular/Templates/UITemplates/ or https://js.devexpress.com/Angular/Demos/WidgetsGallery/Demo/Templates/ (with descriptions) instead of the separate links for  the different themes